workflow/schemas/repository.json

```diff
@@ -1,7 +1,7 @@
 {
-  "title": "Dataset Submission Form",
-  "description": "The repository submission form allows users to submit data to a data repository.",
-  "case_description": "Repository Dataset Submission",
+  "title": "Repository Submission",
+  "description": "The repository submission form allows users to submit files to a data repository.",
```

Comment:
I think you forgot a "data" here?

Reply:
I was just said the consensus was to use "repositories" and "file"?

Reply:
oh